scripts/qa_crowbarsetup.sh

```diff
@@ -3590,7 +3591,15 @@ function oncontroller_magnum_service_setup ()
         # os-distro property
         openstack image create --file $service_image_name \
             --disk-format qcow2 --container-format bare --public \
-            --property os_distro=fedora-atomic magnum-service-image
+            --property os_distro=jeos magnum-service-image
```

Comment:
Why not ```opensuse``` which is listed at https://wiki.openstack.org/wiki/Obsolete:Glance-common-image-properties-os_distro#opensuse ?